scripts/install_evmone.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env sh
+set -e
+
+TEMPDIR="src"
+cd /
+mkdir -p $TEMPDIR
+cd $TEMPDIR
+git clone https://github.com/chfast/evmone.git
+(
+  cd evmone
+  git submodule update --init --recursive
+  mkdir build
+  cd build
+  cmake .. -DBUILD_SHARED_LIBS=OFF
+  make -j2
```

Comment:
How will the binaries be discovered?

Reply:
We only depend on static libraries (no executable binaries are compiled).

The static library link directory will be added to the `ossfuzz`  CMakeLists.txt like so

https://github.com/ethereum/solidity/blob/cf1c236d4945e4937ba5c6848f2ef6ee69e9ae4c/test/tools/ossfuzz/CMakeLists.txt#L1

Reply:
Maybe use the `deps` subdirectory for that?

Reply:
Or do you need a global absolute path?
Note that this has to be run as root on most systems if we do it like that...

Reply:
This was done for oss-fuzz Docker build environment where dependencies would be installed globally, but you are right that it is probably better we move towards a relative directory like `./deps`.

Since moving from global absolute to local relative path would break the oss-fuzz build, shall we treat this as a separate issue?

Reply:
Ok!

Reply:
Created an issue to track this #6845